session-signatures/getLitActionSessionSigs/src/index.ts

```diff
@@ -0,0 +1,103 @@
+import { getEnv } from "./utils";
+import { LitNodeClient } from "@lit-protocol/lit-node-client";
+import {
+  AuthMethodScope,
+  LitNetwork,
+  ProviderType,
+  LIT_RPC,
+} from "@lit-protocol/constants";
+import {
+  EthWalletProvider,
+  LitAuthClient,
+} from "@lit-protocol/lit-auth-client";
+import {
+  LitAbility,
+  LitActionResource,
+  LitPKPResource,
+} from "@lit-protocol/auth-helpers";
+import * as ethers from "ethers";
+import { LocalStorage } from "node-localstorage";
+
+const ETHEREUM_PRIVATE_KEY = getEnv("ETHEREUM_PRIVATE_KEY");
+const LIT_RELAYER_API_KEY = getEnv("LIT_RELAYER_API_KEY");
+
+export const getSessionSigsLitAction = async () => {
+  let litNodeClient: LitNodeClient;
+
+  try {
+    const ethersSigner = new ethers.Wallet(
+      ETHEREUM_PRIVATE_KEY,
+      new ethers.providers.JsonRpcProvider(LIT_RPC.CHRONICLE_YELLOWSTONE)
+    );
+
+    console.log("🔄 Connecting LitNodeClient to Lit network...");
+    litNodeClient = new LitNodeClient({
+      litNetwork: LitNetwork.DatilDev,
+      debug: false,
+      storageProvider: {
+        provider: new LocalStorage("./lit_storage.db"),
+      },
+    });
+    await litNodeClient.connect();
+    console.log("✅ Connected LitNodeClient to Lit network");
+
+    console.log(
+      "🔄 Initializing LitAuthClient for authentication through Lit login..."
+    );
+    const litAuthClient = new LitAuthClient({
+      debug: false,
+      litRelayConfig: {
+        relayApiKey: LIT_RELAYER_API_KEY,
+      },
+      rpcUrl: LIT_RPC.CHRONICLE_YELLOWSTONE,
+      litNodeClient,
+    });
+    console.log("✅ Successfully initialized LitAuthClient instance");
+
+    console.log(
+      "🔄 Authenticating EthWalletProvider, generating an Auth Method..."
+    );
+    const authMethod = await EthWalletProvider.authenticate({
+      signer: ethersSigner,
+      litNodeClient,
+    });
+    console.log(
+      "✅ Successfully authenticated EthWallerProvider, generated Auth Method"
+    );
+
+    console.log("🔄 Minting a PKP using Auth Method...");
+    const pkp = await litAuthClient.mintPKPWithAuthMethods([authMethod], {});
+    console.log("✅ Successfully minted a PKP using the Auth Method");
+
+    const litActionCode = `const go = async () => {
+      Lit.Actions.setResponse({ response: "true" });
+   };
+   
+   go();
+   `;
+    console.log("🔄 Getting Session Sigs...");
+    const sessionSignatures = await litNodeClient.getLitActionSessionSigs({
+      pkpPublicKey: pkp.pkpPublicKey!,
+      authMethods: [authMethod],
+      chain: "ethereum",
+      resourceAbilityRequests: [
+        {
+          resource: new LitPKPResource("*"),
+          ability: LitAbility.PKPSigning,
+        },
+        {
+          resource: new LitActionResource("*"),
+          ability: LitAbility.LitActionExecution,
+        },
+      ],
+      litActionCode: Buffer.from(litActionCode).toString("base64"),
```

Comment:
I'm pretty sure this doesn't have to be encoded and can just be passed as a string (and the SDK will handle encoding)

```suggestion
      litActionCode,
```

Reply:
Did this because of what the SDK says, has this changed?:

The litActionCode is the JavaScript code that will run on the nodes. You will need to convert the string content to base64.

@example

Buffer.from(litActionCodeString).toString('base64');

Reply:
The SDK might handle the formatting for you, because I've always just passed it as a string

Reply:
Just put the string instead